src/aura/STG_CMP_Addr/STG_CMP_Addr.cmp

```diff
@@ -10,6 +10,49 @@
     <aura:attribute name="householdRecTypeId" type="String" />
 
     <div class="slds-grid slds-wrap">
+        <div class="slds-col slds-size--1-of-2">Disable Preferred Email enforcement:</div>
+        <div class="slds-col slds-size--1-of-2">
+            <div class="slds-form-element">
+                <div class="slds-form-element__control">
+                    <label class="slds-checkbox">
+                        <aura:if isTrue="{!v.isView}">
+                            <ui:outputCheckbox value="{!v.hierarchySettings.Disable_Preferred_Email_Enforcement__c}"
+                                               class="contact-addr-enabled" />
+                        <aura:set attribute="else">
+                            <ui:inputCheckbox value="{!v.hierarchySettings.Disable_Preferred_Email_Enforcement__c}"
+                                              class="contact-addr-enabled" />
+                            <span class="slds-checkbox--faux"></span>
+                            <span class="slds-form-element__label"></span>
+                        </aura:set>
+                        </aura:if>
+                    </label>
+                </div>
+            </div>
+        </div>
+        <div class="slds-col slds-size--1-of-1 slds-m-bottom--medium">
+            <aura:if isTrue="{!v.namespacePrefix == 'hed__'}">
+                <ui:outputText value="{!$Label.hed.stgHelpContactPreferredEmail}" class="slds-text-body--small" />
+            <aura:set attribute="else">
+                <ui:outputText value="{!$Label.c.stgHelpContactPreferredEmail}" class="slds-text-body--small" />
+            </aura:set>
+            </aura:if>
+        </div>
+
+        <div class="slds-col slds-size--1-of-2 slds-p-right--xx-large">
+            <h2>Preferred Email Data Clean Up:</h2>
+            <div class="slds-text-body--small">
+                <p>You can run this data clean up process if the Preferred Email enforcement has been disabled for some time, or if the Preferred Email enforcement is being turned on for the first time.</p>
```

Comment:
@davidjray I suggest some changes to the heading and body text.
<h2>Preferred Email Data Cleanup:</h2>
<p>Preferred Email is a required field for Contacts. If you're upgrading from an earlier version of HEDA that didn't enforce this requirement, run this batch cleanup process to make sure that all Contacts have a Preferred Email specified.</p>